docs/source/authentication.rst

```diff
@@ -7,11 +7,8 @@ That means that any authentication method supported by an installed PAS Plugin s
 
 For example, to authenticate using HTTP basic auth, you'd set an ``Authorization`` header:
 
-.. code-block:: http
-
-  GET /Plone HTTP/1.1
-  Authorization: Basic Zm9vYmFyOmZvb2Jhcgo=
-  Accept: application/json
+.. literalinclude:: _json/siteroot.req
+   :language: http
```

Comment:
Here instead of using the `..literalinclude` I would use a `.. http:example` to show how you authenticate using several tools (requests, httpie, curl, ...)